README.md

```diff
@@ -17,24 +17,12 @@ To use the Sass rules, add the following to your
 published versions:
 
 ```py
-# To use the JavaScript version of Sass, we need to first install nodejs
-http_archive(
-    name = "build_bazel_rules_nodejs",
-    url = "https://github.com/bazelbuild/rules_nodejs/archive/0.8.0.zip",
-    strip_prefix = "rules_nodejs-0.8.0",
-    sha256 = "4e40dd49ae7668d245c3107645f2a138660fcfd975b9310b91eda13f0c973953",
-)
-load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
-node_repositories(package_json = [])
-
-http_archive(
```

Comment:
Why doesn't the user need to write this any more?

Reply:
I accidentally force pushed my online `README.md` edit. will fix.